test/testcafe/spec/UnlockAccountChallenge_spec.js

```diff
@@ -48,7 +48,7 @@ test.requestHooks(identifyLockedUserMock)('should show unlock account link', asy
 });
 
 
-test.requestHooks(identifyLockedUserMock)('should render custom Unlock account link', async t => {
+test.meta('v3', false).requestHooks(identifyLockedUserMock)('should render custom Unlock account link', async t => {
```

Comment:
had to disable this test since v3 does not currently support custom unlock account links.